### DISCUSSION TITLE
How to see if a device it's online using Cloud

### DISCUSSION THREAD

**Discussion by flash370** [jasonacox/tinytuya#233] [Q&A]
Hi,

I am searching to see if a device it's online or offline using the cloud methods.
In the JSON reply I don't found anything.
Have you some suggestion?

Now I am using these functions:
devices = c.getdevices()
result = c.getproperties(id)
result = c.getstatus(id)

The only function that replays that a device is offline is:
c.sendcommand(id,commands)

BUT, I don't want send commands to see if it's online........

Thanks

**Comment by uzlonewolf** [comment]
```python
devices = c.getdevices(verbose=True)

if devices and 'result' in devices:
    for device in devices['result']:
        print( device['id'], 'online:', device['online'] )
```

**Reply by flash370** [reply]
Thanks!

It's pretty easy!! ;D
